extensions/jackson/runtime/src/main/java/io/quarkus/jackson/runtime/JacksonBuildTimeConfig.java

```diff
@@ -43,4 +45,10 @@
      */
     @ConfigItem(defaultValue = "UTC")
     public Optional<ZoneId> timezone;
+
+    /**
+     * Define which properties of Java Beans are to be included in serialization
```

Comment:
```suggestion
     * Define which properties of Java Beans are to be included in serialization.
```
Also, is doc generation able to cope with the Jackson class? There is a nice link to `ZoneId` (for `timezone`) but then again that's a JDK class...?

Reply:
The doc generation does won't be able to cope with it.

Reply:
Actually the doc generation works:

```asciidoc
a|icon:lock[title=Fixed at build time] [[quarkus-jackson-jackson-build-time-config_quarkus.jackson.serialization-inclusion]]`link:#quarkus-jackson-jackson-build-time-config_quarkus.jackson.serialization-inclusion[quarkus.jackson.serialization-inclusion]`

[.description]
--
Define which properties of Java Beans are to be included in serialization.
--|`always`, `non-null`, `non-absent`, `non-empty`, `non-default`, `custom`, `use-defaults` 
|
```

Reply:
> Actually the doc generation only displays the possible enum values:
> 
> ![image](https://user-images.githubusercontent.com/54133/129651850-36c68ed9-140d-48d4-8100-522de62ca592.png)

Yes, if the underlying value is an enum, they'll be enlisted. If it is a JDK type or some known type a link to a javadoc is added, otherwise nothing.

Reply:
My point is to give users a link to the jackson javadoc, so that they can look up what those values mean. If the generation cannot do it we should add a link ourselves...?
At the very least there should be a note on the FQCN so that users can look it up themselves.

Reply:
> If the generation cannot do it we should add a link ourselves...?

For enums type we thought it is of much value to enlist possible enum values in hyphenated format. Users can see what value they can put and use them. 

But for other types, yes if we find that a link is of value they it can be added.